practice-app/event_app/views.py

```diff
@@ -91,3 +91,15 @@ def viewGithubInfo(req):
     except Exception as e:
         print(str(e))
         return HttpResponseRedirect('../boun/viewGithubInfoPage?fail=true')
+
+def viewActivity(req):
+
+    url = "https://www.boredapi.com/api/activity" 
+
+    resp = requests.get(url)
```

Comment:
I think we should add a try except block here, to ensure that api will give a response.